gateway-service/src/main/resources/application.yml

```diff
@@ -122,7 +122,8 @@ zuul:
     includeDebugHeader: false
     sensitiveHeaders: Expires,Date
     ignoredPatterns:
-        - /**/ws/**
+        - /ws/**
```

Comment:
Use /**/ws/** and /**/sse/**/ - we need to support patterns that doesn't start with the part of the URL.

Reply:
Suggested change made. Thanks